sound/soc/sof/intel/hda-mlink.c

```diff
@@ -756,6 +771,35 @@ int hdac_bus_eml_sdw_set_lsdiid(struct hdac_bus *bus, int sublink, int dev_num)
 	return 0;
 } EXPORT_SYMBOL_NS(hdac_bus_eml_sdw_set_lsdiid, SND_SOC_SOF_HDA_MLINK);
 
+int hdac_bus_eml_sdw_map_stream_ch(struct hdac_bus *bus, int sublink, int y,
```

Comment:
what is the `y` parameter?

Reply:
it's named after the register: PCMSyCM.

y is the PDI number in this case.

I could add comments to make it self-explanatory.

Reply:
added comments and macro